package/Linux.mk

```diff
@@ -98,7 +98,9 @@ install:
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/applications
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/pixmaps
-	$(V1) $(MKDIR) -p $(DESTDIR)$(udevdir)
+# Following line creates udev directory - however udevdir is not previously defined, nor is anything
+# installed to this location in the rest of this recipe
+#	$(V1) $(MKDIR)  $(DESTDIR)$(udevdir)
```

Comment:
Thanks for that. The issue which has prevented me writing the ebuild previously is the potential dependency issues with the build environment, and the target system. In fact the installation script puts all the files in the right place, but I need to make sure I can get the ebuild to call in the tools according to "make all_sdk_install". I am sure it can be done - I just need to work it out!